products/firefox_android/adding_multilocales.md

```diff
@@ -28,15 +28,38 @@ $ hg pull -r default -u
 $ hg update central
 ```
 
-The file to modify is in `mobile/android/locales/maemo-locales`, open it with your text editor of choice.
+The first file to modify is in `mobile/android/locales/maemo-locales`, open it with your text editor of choice.
 
 ```BASH
 $ atom mobile/android/locales/maemo-locales
 ```
 
 And add the new locale to the list. With Atom and the Sort Lines package installed, you can press `F5` to make sure that the list is in **alphabetical order**. Let’s say for example that you need to add `ab-CD` to the list of supported locales.
 
-After you’ve finished editing the file, check the status of the repository, and the diff.
+The second file to modify is `mobile/android/locales/l10n.toml`. This is the beginning of the file:
+
+```
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+basepath = "../../.."
+
+locales = [
+    "ab-CD",
+    "an",
+    "ar",
+    "as",
+    "ast",
+    "az",
+...
+```
+
+Identify the `locales` section, and make sure the locale is already listed under the `locales` section (it should be already). In the example above, the locale is represented by `ab-CD`.
```

Comment:
Maybe: it should be already -> it should have been added when enabling single locale builds?

Reply:
Agreed, sounds better.